src/test/java/com/google/devtools/build/lib/remote/RemoteExecutionServiceTest.java

```diff
@@ -2352,7 +2365,9 @@ public void buildRemoteActionForRemotePersistentWorkers() throws Exception {
                     Platform.Property.newBuilder()
                         .setName("persistentWorkerKey")
                         .setValue(
-                            "628637504c26bb74fb6bb3f60fb7132b3aa574b866db4181770774882a8853e5"))
+                            enablePathMapping
+                                ? "85e3ad12f36ccb7b5eddbfe8f6bc28f57004634c537faac32d33a30b8d456bb8"
```

Comment:
Is it possible to compute these hashes from the respective paths? In particular, we need to take into account the product name (outputs paths will be `blaze-out/...` instead of `bazel-out/...` when we run these tests internally)

Reply:
Actually, nevermind; it doesn't look like the paths in this test contain the product name (the tests are failing internally but it must be some other reason - let me take a look)

Reply:
Oh, it's the `TestConstants.WORKSPACE_NAME` (it's `google3` instead of `_main` internally).

I guess there isn't a better way to do this than switch on the workspace name, since computing the `persistentWorkerKey` using the production logic defeats the point of the test?